### ISSUE TITLE
Device tracking does not work anymore

### ISSUE THREAD

**Issue by markusspa** [home-assistant/iOS#2583]
Model Name:
iPhone 14 Pro & iPhone 12 Pro

Software Version:
iOS 17.3.1

App version: 
Home Assistant Companion App 2024.1 (2024.535)

Home Assistant Core Version:
2024.2.1

Description of the bug:
The location of my device is not updating anymore. That is why I am always home although I left the home zone. I checked all preferences in iOS settings as well as the settings of the HA App (Location, Background activities, ...). VPN works well. My girlfriend has exactly the same problem.

The activity protocol in the app shows a lot of errors. Perhaps it helps to solve the problem. The errors are always the same:
- Webhook failed with status code 503
- Didn't update [...] (Schared.BackgroundTaskError-Fehler 0.)

**Comment by bgoncal** [comment]
Was this working at some point with the same VPN setup? When do you connect to the VPN?
As a possible quick fix can you reboot your iphone?

**Comment by markusspa** [comment]
Yes, it was. The VPN setup did not change and it works. When I am away, I have access to Home Assistant, dashboards and I am able to control everything. But the location is not updating. A restart of my phone was not successful. I know from others who have the same problem, too (Android and iOS).

**Comment by NielsAutomation** [comment]
Same problem here (also IOS). My setup is without a VPN.

**Comment by bgoncal** [comment]
Checkout location history and share what you see there please. Also you can enable notifications for when you enter and exite zones, if you do that, let me know how it goes.
https://github.com/home-assistant/iOS/assets/5808343/d78c2056-0b40-45bc-abb1-3be6914f26c3

**Comment by NielsAutomation** [comment]
It looks like I solved it on one of the iPhones. After some digging, I saw an error in the companion app (Webhook failed with status code 503). After deleting the server from the app and reconnecting it it worked again.

For the people who don't know where you can find the logs from the companion app:
- scroll down in the app and click on problem solving;
- click on event log.

**Comment by markusspa** [comment]
Thanks @NielsAutomation! It seems to work for me, too.

**Comment by alexwhittemore** [comment]
This happened to me after the latest update. Only if I delete my server and try to re-add it, I see this:

![image](https://github.com/home-assistant/iOS/assets/332827/42571402-59b5-431f-8500-d3ead3d21969)

**Comment by the-misterlas** [comment]
Also seeing an issue with multiple iOS device locations not updating. 

iOS 17.3.1
HA 2024.2.2

**Comment by koburg** [comment]
Yes me to two iPhones 16 Pro also not updating this has been working flawless for long time but not in last 2-3 months

**Comment by bgoncal** [comment]
Check your logs, if you have webhook 503 error, please delete the mobile app integration or delete the server in HA app and add it back

**Comment by igotsoul** [comment]
+ 1, experienced the same. Some attributes are updated, some ain't. Location remains "at home" for 24h now, even though zone has been left. Checked all the potential issues that are described above likes of app permission for localization etc. No success.

**Comment by bgoncal** [comment]
Please try this first https://github.com/home-assistant/iOS/issues/3332

**Comment by igotsoul** [comment]
Hi @bgoncal 

Yeah, I just came across it while you were replying. I think I might be misunderstanding something. Do I need an active remote connection to update my location when leaving my home zone and disconnecting from my Wi-Fi? That could be the issue since, if I’m not mistaken, the location update isn’t done via iCloud but directly to the Home Assistant server. Is that correct?

**Comment by bgoncal** [comment]
@igotsoul Correct, the app has to have a way to access your home assistant server back in your home, and this way is through remote access, without that the app does not know where to send the information to

**Comment by igotsoul** [comment]
@bgoncal That makes complete sense. In the meantime, I’ve configured it and will give it another try. Apologies for the misunderstanding, and a big shoutout for your quick assistance!
